src/helpers/index.js

```diff
@@ -17,11 +22,11 @@ export function captureField(e: { target: { id: string, value: string}}, trigger
     default:
       console.log('This is the default case');
   }
-}
+};
 
-export function submitForm(e: { preventDefault: Function }, values: { name: string, email: string, company: string, phone: string }) {
+export const submitForm = (e: PreventDefaultFunction, values: Values) => {
```

Comment:
This preventDefaultFunction is too explicit for its use. PreventDefault is part of the event object. In this case you should type 'Event' and include all your uses. This allows it to be scalable for future use and you impact one set of code vs needing a major refactor to simplify later.